docs/product/searcher.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: developer-doc
```

Comment:
I did not notice it earlier - please note that all of our docs should have 80 chars limit. We made change toallow 100 chars limit for Rust code, but docs across the wholeproject should be formatted according to the rulesin main enso repo, which specifies80 chars limit